src/types/exports.ts

```diff
@@ -80,14 +80,17 @@ export interface EdgeContextProps extends CommonProps {
   apiKey?: string
   apiSecret?: Uint8Array
   appId?: string
-  authServer?: string
   infoServer?: string | string[]
+  loginServer: string | string[] // Do not include `/api` in the path
```

Comment:
JSDoc style comment `/** Do not include `/api` in the path */` for editor friendliness?